site/blog/2016/jul/1/sphinx-and-rtd-for-writers.rst

```diff
@@ -0,0 +1,548 @@
+Open Source in this context means both that the code is open source,
+and the documentation hosted is for open source software.
+It is a developer focused platform,
+but it has most of the features that technical writers have come to expect in a tool as well.
+This blending of worlds makes it the best suited platform for teams that want both writers and developers contributing to product and API documentation.
+
+Read the Docs is best thought of as a continuous documentation platform for Sphinx.
+Continuous documentation being similar to continuous integration in programming circles.
+In practice it means that your documentation is built,
+tested, 
+and updated on every commit.
+
+Sphinx provides a documentation generator that is best-in-class for software docs.
+Sphinx documents are written in the reStructuredText markup language.
+which is similar to Markdown,
+but much more powerful.
+Read the Docs provides a best-in-class hosting platform for Sphinx.
+
+As a writer who uses Sphinx,
+your day to day will be writing reStructuredText in text files.
+You will then build your documentation by running Sphinx on the command line.
+Generally it's easiest to output HTML for local writing and testing,
+and then you can let Read the Docs generate PDF's and other formats.
+
+This article will provide an overview of the features of Sphinx and Read the Docs,
+and allow you to evaluate them for use in your organization.
+
+Introduction to Sphinx
+----------------------
+
+`Sphinx <http://www.sphinx-doc.org/>`_ is the documentation tool of choice for the Python language community,
+and increasingly large parts of the software ecosystem.
+It was `originally created <https://en.wikipedia.org/wiki/Sphinx_(documentation_generator)>`_ in 2008 to document the Python language itself.
+
+Over the past eight years,
+it has turned into a robust and mature solution for software documentation.
+It has a number of features that technical writers would expect,
+including:
+
+* Single Source Publishing 
+    * Output HTML, PDF, ePub, and more
+    * Content reuse through includes
+    * Conditional includes based on content type and tags
+* Multiple mature HTML themes that look beautiful on mobile and desktop
+* Referencing across pages, documents, and projects
+* Index and Glossary support
+* Internationalization support
+
+It also has practical and powerful tools for documenting software specifically,
+including:
+
+* Semantic referencing of software concepts, including Classes, Functions, Programs, Variables, etc.
+* Including of code comments inside prose documentation for many programming languages
+* Tools for documenting HTTP API's
+* Extensions with the Python language
+* A vast array of third party extensions to do useful things
+
+This article isn't large enough to cover all of the power packed into this tool.
+Hopefully I can show you enough that it piques your interest and you can research more.
```

Comment:
But I hope to show enough to pique reader interest so that you can try these tools out and research their capacity for yourself.